src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -50,7 +50,12 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
+        if (person.getAddress() != null) {
+            Label address = new Label(person.getAddress().value);
```

Comment:
maybe can write a `hasAddress()` method on `Person` class that just `return address.isPresent()`?

Reply:
alternatively, if you change `getAddress()` to return an `Optional<Address>`, then you can chain a `person.getAddress().isPresent()`